schema/data-product-definition-schema.json

```diff
@@ -106,6 +114,53 @@
       "additionalProperties": true
     },
 
+    "dashboard": {
+      "type": "object",
+      "required": ["id", "name", "description", "metrics"],
+      "id": "string",
+      "name": "string",
+      "description": "string",
+      "metrics": {
+        "type": "array",
+        "items": { "$ref": "#/definitions/metric-definition-summary" }
+      }
+    },
+
+    "metric-definition-summary": {
+      "type": "object",
+      "required": ["id", "visualisationType"],
+      "id": "string",
+      "visualisationType":  {
+        "type": "array",
+        "items": { "type": "string" }
+      }
```

Comment:
How is the summary related to the metric? Dows the summary ID match the metric ID?

Reply:
The idea was the the dashboard will contain only a summarised version of the metric without all the fields and the metric-summary (I think might rename it to dashboard metric as this is the way I have named it in the [lib](https://github.com/ministryofjustice/hmpps-digital-prison-reporting-lib/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/digitalprisonreportinglib/data/model/Dashboard.kt#L9)) will be linked to the full metric with their ids being the same.

` "dashboards": [
    {
      "id": "test-dashboard-1",
      "name": "Test Dashboard 1",
      "description": "Test Dashboard 1 Description",
      "metrics": [
        {
          "id": "missing-ethnicity-metric",
          "visualisationType": ["bar"]
        }
      ]
    }
  ],
  "metrics": [
    {
      "id": "missing-ethnicity-metric",
      "name": "Missing Ethnicity By Establishment Metric",
      "dataset": "missing-ethnicity-dataset",
      "display": "Missing Ethnicity By Establishment Metric",
      "description": "Missing Ethnicity By Establishment Metric",
      "visualisationType": ["bar", "doughnut"],
      "specification": [
        {
          "name": "establishment_id",
          "display": "Establishment ID"
        },
        {
          "name": "missing_ethnicity_percentage",
          "display": "Missing Ethnicity Percentage",
          "unit": "percentage"
        },
        {
          "name": "present_ethnicity_percentage",
          "display": "Present Ethnicity Percentage",
          "unit": "percentage"
        }
      ]
    }
  ]`

Reply:
I have just renamed the `metric-definition-summary` to  `dashboard-metric-definition`